include/jemalloc/internal/hpa_central.h

```diff
@@ -0,0 +1,41 @@
+#ifndef JEMALLOC_INTERNAL_HPA_CENTRAL_H
```

Comment:
Similar to my comment in #1904, it would be great if you could write some design doc on how these APIs are supposed to be used in the HPA world.